tsconfig.json

```diff
@@ -8,7 +8,7 @@
     "esModuleInterop": true,
     "noUnusedLocals": true,
     "lib": ["ESNext", "DOM"],
-    "types": ["vite/client"],
+    "skipLibCheck": true,
```

Comment:
Question: may I know why we modified those tsconfig options?
The else part looks good to me btw.

Reply:
vite isn't there as a dep/dev-dep in package.json. Its a transitive dep (via vitepress). Since we are using pnpm, tsc won't be able to pick those files anyway.  We can keep it though, but it will be redundant.